.github/workflows/build-master-blue-arm64.yaml

```diff
@@ -62,10 +62,18 @@ jobs:
       - name: Run make deps_ci
         run: |
           export DOCKER_INSTALL=true
+          export DOCKER_INSTALL=false
```

Comment:
duplicated entry here too int he final output

Reply:
argh there it's a bit not nice from the template itself, there the first one is coming from another option, not sure if want to make it too tight to the override